cluster-scope/base/user.openshift.io/groups/ttm-as-a-service/group.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: ttm-as-a-service
+users:
+  - Gkrumbach07
+  - gkrumbach07
+  - harshad16
+  - goern
+  - kpostoffice
+  - KPostOffice
```

Comment:
only appropriate cased usernames are needed, you can remove the lower-case duplicates

Reply:
done